wearApp/src/main/java/dev/johnoreilly/confetti/wear/sessiondetails/SessionDetailsView.kt

```diff
@@ -30,59 +32,91 @@ import java.time.ZoneOffset
 @Composable
 fun SessionDetailsRoute(
     columnState: ScalingLazyColumnState,
-    onBackClick: () -> Unit,
     viewModel: SessionDetailsViewModel = getViewModel()
 ) {
     val session by viewModel.session.collectAsStateWithLifecycle()
-    SessionDetailView(session, columnState, onBackClick)
+    val timeZone = remember { viewModel.timeZone }
+    SessionDetailView(
+        session = session,
+        columnState = columnState,
+        formatter = { viewModel.formatter.format(it, timeZone, "eeee HH:mm")})
 }
 
 @Composable
 fun SessionDetailView(
     session: SessionDetails?,
     columnState: ScalingLazyColumnState,
-    popBack: () -> Unit
+    formatter: (Instant) -> String
 ) {
-    val context = LocalContext.current
+    val description = session.descriptionParagraphs()
 
     ScalingLazyColumn(columnState = columnState) {
         session?.let { session ->
             item {
-                ListHeader {
+                Row(
+                    modifier = Modifier.padding(horizontal = 20.dp)
+                ) {
                     Text(
                         text = session.title,
+                        color = MaterialTheme.colors.onSurfaceVariant,
+                        style = MaterialTheme.typography.button
                     )
                 }
             }
 
-            session.description?.let {
-                item {
-                    Text(text = it)
+            item {
+                val time = remember(session) {
+                    formatter(session.startInstant)
                 }
+                Text(time)
             }
 
-//            if (session.tags.isNotEmpty()) {
-//                Spacer(modifier = Modifier.size(16.dp))
-//                    FlowRow(crossAxisSpacing = 8.dp) {
-//                        session.tags.forEach { tag ->
-//                            Chip(tag)
-//                        }
-//                    }
-//            }
+            items(description) {
+                Text(text = it)
+            }
 
             items(session.speakers) { speaker ->
-                SessionSpeakerInfo(speaker = speaker.speakerDetails,
-                    onSocialLinkClick = { socialItem, _ ->
-                        val intent = Intent(Intent.ACTION_VIEW, Uri.parse(socialItem.link))
-                        context.startActivity(intent)
-                    }
-                )
+                SessionSpeakerInfo(speaker = speaker.speakerDetails)
             }
         }
     }
 }
 
-@RequiresApi(Build.VERSION_CODES.O)
+private fun SessionDetails?.descriptionParagraphs(): List<String> {
```

Comment:
@joreilly any thoughts on how to handle this? During the load?

Reply:
@martinbonnin I think you were hoping to "santize" this data at some point? (this is specific I think to fosdem info?)

Reply:
Indeed, this is only for Fosdem data that is HTML. I can move that logic to the backend

Reply:
https://github.com/joreilly/Confetti/issues/199